python_modules/dagster/dagster/_core/run_coordinator/default_run_coordinator.py

```diff
@@ -50,4 +50,10 @@ def submit_run(self, context: SubmitRunContext) -> DagsterRun:
         return run
 
     def cancel_run(self, run_id: str) -> bool:
+        run = self._instance.get_run_by_id(run_id)
+        if run is None:
+            check.failed(f"Failed to load run {run_id}")
+        self._instance.report_run_canceling(
+            run, message="Received cancellation request from default run coordinator"
```

Comment:
I would suggest dropping "from default run coordinator" - don't think a user needs to understand what that is